docs/roles/install_nautobot.rst

```diff
@@ -0,0 +1,104 @@
+
+Required Variables
+------------------
+
+The variables required need to be defined in the inventory or the
+playbook for the host.
+
+==================== ======= ==================================================================
+Variable Name        Default Description
+==================== ======= ==================================================================
+cert_country                 Country for the certificate
+cert_state                   State for the certificate
+cert_city                    City for the certificate
+cert_domain                  Domain for the certificate
+nautobot_db_username         Username for the Nautobot database
+nautobot_db_password         Password for the Nautobot database
+nautobot_db_name             Name of the Nautobot database
+redis_password               Password for the redis database
+allowed_hosts                Comma separated list of hosts for the Django Allowed Hosts setting
+==================== ======= ==================================================================
+
+Optional Variables
+------------------
+
+These variables control the operations of the playbook.
+
++------------------------------------------+----------+----------------+
+| Variable Name                            | Default  | Description    |
++==========================================+==========+================+
+| create_new_self_signed_cert              | False    | Boolean on     |
+|                                          |          | whether or not |
+|                                          |          | should create  |
+|                                          |          | a new self     |
+|                                          |          | signed cert    |
+|                                          |          | when one       |
+|                                          |          | exists         |
+|                                          |          | already.       |
++------------------------------------------+----------+----------------+
+| nautobot_http_port                       | “80”     | HTTP port to   |
+|                                          |          | listen on      |
++------------------------------------------+----------+----------------+
+| nautobot_https_port                      | “443”    | HTTPS port to  |
+|                                          |          | listen on      |
++------------------------------------------+----------+----------------+
+
+Gathered Variables in Playbook
+------------------------------
+
+These are the variables that are defined in the playbook execution.
+These should be set if running your own playbook with the roles that are
+provided.
+
++------------------------------+--------------+-----------------------+
+| Variable Name                | Default      | Description           |
++==============================+==============+=======================+
+| os_family                    | ansible      | lower                 |
```

Comment:
description should be updated to read "lower case name of os family/distribution redhat or debian